lib/ansible/modules/system/sysvinit.py

```diff
@@ -310,8 +310,8 @@ def runme(doit):
                 (rc, out, err) = module.run_command(cmd)
             # FIXME: ERRORS
 
-            if rc != 0:
-                module.fail_json(msg="Failed to %s service: %s" % (action, name))
+            if err != "":
+                module.fail_json(msg="Failed to %s service: %s" % (action, name), rc=rc, out=out, err=err)
```

Comment:
i would uset stdout and stderr as these are how we commonly name these fields in other modules.